tests/Doctrine/Tests/ORM/Functional/ExtraLazyCollectionTest.php

```diff
@@ -1146,6 +1146,7 @@ public function testRemoveOrphanedManagedElementFromOneToManyExtraLazyCollection
 
         $user->userLists->removeElement($this->_em->find(UserList::class, $userListId));
 
+        $this->_em->flush();
```

Comment:
Yes, but it's still a BC break ;-)

I'm fine with documenting it after we looked at what scenarios may be affected by it.